packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -12,30 +12,30 @@ This guide explains how to build transactional verification using a Client-Initi
-* A configured [mobile app that verifies user identities for an Okta custom authenticator](/docs/guides/authenticators-custom-authenticator/ios/main/) and [responds to CIBA authorization challenges](https://github.com/okta/okta-devices-swift#enable-using-your-app-for-client-initiated-backchannel-authentication-ciba) sent by Okta in a backchannel request.
-* A test user in your org that you can use for testing the CIBA flow with a custom authenticator.
+* A configured [mobile app that verifies user identities for an Okta Custom Authenticator](/docs/guides/authenticators-custom-authenticator/ios/main/) and [responds to CIBA authorization challenges](https://github.com/okta/okta-devices-swift#enable-using-your-app-for-client-initiated-backchannel-authentication-ciba) sent by Okta in a backchannel request.
+* A test user in your org that you can use for testing the CIBA flow with a Custom Authenticator.
 * The CIBA feature enabled for your org. From the left navigation pane in the Admin Console, go to **Settings** > **Features**, locate the Transaction Verification feature and slide to enable.
 
 > **Note:** See the [Okta Devices Swift SDK](https://github.com/okta/okta-devices-swift) for iOS and the [Okta Mobile Kotlin SDK](https://github.com/okta/okta-mobile-kotlin) for Android for more information on the Okta Devices SDK. This guide uses the [Okta Devices Swift SDK](https://github.com/okta/okta-devices-swift#okta-devices-sdk).
 
 **Sample code**
 
-This guide uses the [Okta Authenticator Sample App](https://github.com/okta/okta-devices-swift/tree/master/Examples/PushSampleApp) for iOS to configure the Okta custom authenticator to work with CIBA.
+This guide uses the [Okta Authenticator Sample App](https://github.com/okta/okta-devices-swift/tree/master/Examples/PushSampleApp) for iOS to configure the Okta Custom Authenticator to work with CIBA.
 
 ---
 
-## About CIBA
+## About Client-Initiated Backchannel Authentication (CIBA)
 
 Organizations are constantly looking for ways to strike a balance between offering a frictionless user experience without compromising security. It becomes even more challenging when the users try to perform sensitive transactions. Okta uses CIBA to provide customers with a simple and secure transaction verification solution.
 
-CIBA extends OIDC to define a decoupled flow where the authentication or transaction flow is initiated on one device and verified on another. The device in which the transaction is initiated by the OIDC application is called the consumption device and the device where the user verifies the transaction is called the authentication device.
+CIBA extends OIDC to define a decoupled flow where the authentication or transaction flow is initiated on one device and verified on another. The device in which the OIDC application initiates the transaction is called the consumption device and the device where the user verifies the transaction is called the authentication device.
```

Comment:
would it be the 'device on which' instead of 'in which'? I usually say I'm running an app on my device.